images/base-notebook/start-notebook.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+import os
+import shlex
+import sys
+
+# If we are in a JupyterHub, we pass on to `start-singleuser.py` instead so it does the right thing
```

Comment:
I appreciate it that you added comments to this file! I'm sure it will help future contributors to understand this file faster.

Reply:
@mathbunnyru you're welcome :)